recipes/psycopg2-yugabytedb/meta.yaml

```
@@ -0,0 +1,48 @@
+{% set name = "psycopg2-yugabytedb" %}
+{% set version = "2.9.3.post0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/psycopg2-yugabytedb-{{ version }}.tar.gz
+  sha256: e77d230f8e754c8d0940834314d49af80ae6a801560deb52eae5f3fa4155c7ba
+
+build:
+  skip: true  # [py2k]
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - python                                 # [build_platform != target_platform]
+    - cross-python_{{ target_platform }}     # [build_platform != target_platform]
+    # libpq is a repackaging of only the library to connect to postgres
+    - libpq >=14.0,<17.0
+  host:
+    - python
+    - pip
+    - libpq >=14.0,<17.0
```

Don't add version constraints so that it uses the current global pin.
```suggestion
    - libpq
```